sdk/core/Azure.Core.TestFramework/src/PlaybackTransport.cs

```diff
@@ -121,6 +121,12 @@ public Response GetResponse(RecordEntry recordEntry)
                 {
                     response.AddHeader(new HttpHeader(responseHeader.Key, value));
                 }
+
+                // Ignore the Retry-After header wait time for 429 responses during playback.
+                if (response.Status == 429)
+                {
+                    response.AddHeader(new HttpHeader("Retry-After", "0"));
+                }
```

Comment:
This works nicely, but seems wrong. Thoughts?

Reply:
Why are we adding this?

Reply:
Ah nevermind - nice comment..

Reply:
Should we be adding the header or updating it?

Reply:
Updating would be ideal.

Reply:
Changed to  update.